handlers/network_data_aggregator.py

```diff
@@ -0,0 +1,12 @@
+import redis
+import json
+
+redis_client = redis.Redis(host='localhost', port=6379, db=0)
+
+def handle(node_status: dict, _context):
+
+    redis_client.set(node_status['server']['id'], json.dumps(node_status))
```

Comment:
How are we sure both keys will exist?

Reply:
Keys will be abstracted by `CacheClient` https://github.com/HathorNetwork/hathor-explorer-service/pull/3/files#diff-2d00b3a1edf0e7d331f8615b554640a3ff3892b5fb14b30d0f623319abaf9d2aR41

`config.redis_key_prefix` is the first part. https://github.com/HathorNetwork/hathor-explorer-service/pull/3/files#diff-076cb2d0c23fedf9e506a0465551e1ad7409907323037b0043e22434e278bb39R11